src/test/java/tests/BaseTest.java

```diff
@@ -23,15 +25,17 @@
     /***
      * Set and open the local thread browser type.
      */
-    @BeforeMethod
+    @BeforeMethod(alwaysRun = true)
     public void setUp() {
-        DriverManager.setWebDriver(DriverFactory.getDriver(BrowserType.CHROME));
+        WebDriver driver = DriverFactory.getDriver(BrowserConfig.getType());
```

Comment:
it's better not to create a variable for this but to pass DriverFactory.getDriver(BrowserConfig.getType()) to setWebDriver